### DISCUSSION TITLE
get an error

### DISCUSSION THREAD

**Discussion by Lucascoco** [qdm12/ddns-updater#348] [Q&A]
Hello,
when i start my docker container, i get an error

```
2022/08/10 16:30:10 ERROR json: cannot unmarshal number into Go value of type json.dataModel
```

can someone help?

**Comment by qdm12** [comment]
Your JSON content in your `data/updates.json` is probably malformed.

json.dataModel is specified there:

https://github.com/qdm12/ddns-updater/blob/1ff629dc17851cca9974c71a372dd4e61589fb6e/internal/persistence/json/models.go#L9-L11


Make sure you have your JSON as

```json
{"records": [
  // ...
]}
```

**Reply by Lucascoco** [reply]
where can i find, what i must fill in the updates.json?

**Reply by qdm12** [reply]
In your bind mounted directory (/data in the container). I don't know how you configured the container so feel free to share your config if you can't find it.

**Reply by Lucascoco** [reply]
I run the docker over portianer
i greate a volume for data
in the data folder i greate a config,json
{
  "settings": [
    {
      "provider": "strato",
      "domain": "xxxxx",
      "host": "@",
      "password": "xxxxx",
      "ip_version": "ipv4",
      "provider_ip": true
    }
  ]
}
i great a update,json but i dont now how to fill this
